mne_bids_pipeline/steps/preprocessing/_03_maxfilter.py

```diff
@@ -43,6 +45,139 @@
 from ..._run import failsafe_run, save_logs, _update_for_splits, _prep_out_files
 
 
+# %% eSSS
+def get_input_fnames_esss(
+    *,
+    cfg: SimpleNamespace,
+    subject: str,
+    session: Optional[str],
+) -> dict:
+    kwargs = dict(
+        cfg=cfg,
+        subject=subject,
+        session=session,
+    )
+    in_files = _get_run_rest_noise_path(
+        run=None,
+        task="noise",
+        kind="orig",
+        mf_reference_run=cfg.mf_reference_run,
+        **kwargs,
+    )
+    in_files.update(_get_mf_reference_run_path(add_bads=True, **kwargs))
+    return in_files
+
+
+@failsafe_run(
+    get_input_fnames=get_input_fnames_esss,
+)
+def compute_esss_proj(
+    *,
+    cfg: SimpleNamespace,
+    exec_params: SimpleNamespace,
+    subject: str,
+    session: Optional[str],
+    in_files: dict,
+) -> dict:
+    import matplotlib.pyplot as plt
+
+    run, task = None, "noise"
+    in_key = f"raw_task-{task}_run-{run}"
+    bids_path_in = in_files.pop(in_key)
+    bids_path_bads_in = in_files.pop(f"{in_key}-bads", None)  # noqa
```

Comment:
Because eventually we *should* use this (with the given variable filename) but we have an unaddressed bug. So to me it helps to have the variable name we should be using here, and it commented out (with that same name) below to at least establish the link (which you did implicitly already, even if you didn't understand the "why aren't we actually using it" part!)